spec/lib/schedule_spec.rb

```diff
@@ -51,19 +51,32 @@
       subject.dates(start_date).first.should == start_date + 4 # Saturday
 
       start_date = Date.new(2014, 8, 20) # Wednesday
-      subject.dates(start_date).first.should == start_date + 6 # Tuesday
+      subject.dates(start_date).first.should == start_date + 7 # Wednesday
 
       start_date = Date.new(2014, 8, 21) # Thursday
-      subject.dates(start_date).first.should == start_date + 6 # Wednesday
+      subject.dates(start_date).first.should == start_date + 7 # Thursday
 
       start_date = Date.new(2014, 8, 22) # Friday
-      subject.dates(start_date).first.should == start_date + 6 # Thursday
+      subject.dates(start_date).first.should == start_date + 7 # Friday
 
       start_date = Date.new(2014, 8, 23) # Saturday
-      subject.dates(start_date).first.should == start_date + 5 # Thursday
+      subject.dates(start_date).first.should == start_date + 6 # Friday
 
       start_date = Date.new(2014, 8, 24) # Sunday
-      subject.dates(start_date).first.should == start_date + 4 # Thursday
+      subject.dates(start_date).first.should == start_date + 5 # Friday
+    end
+
+    it "assumes bookings are not processed on bank holidays" do
+      #      April 2015
+      # Su Mo Tu We Th Fr Sa
+      #           1  2 <3> 4
+      #  5 <6> 7  8  9 10 11
+      # 12 13 14 15 16 17 18
+      # 19 20 21 22 23 24 25
+      # 26 27 28 29 30
+
+      start_date = Date.new(2015, 4, 2)
+      subject.dates(start_date).first.should == start_date + 8
     end
```

Comment:
I'm happy with the JSON—as you say, I think they'd think twice about changing it—but we should have a test to make sure we don't get caught out if they do.

The current test should still use mocks just in case the current data we rely on is removed from the JSON: I don't want to have to update the test if the data changes.